app/controllers/renewal/InvolvedInOtherController.scala

```diff
@@ -54,25 +54,19 @@ class InvolvedInOtherController @Inject()(
         }
   }
 
-  private def businessTypes(activities: BusinessMatching): Option[String] = {
-    val typesString = activities.activities map { a =>
-      a.businessActivities.map {
-        case AccountancyServices => Messages("businessmatching.registerservices.servicename.lbl.01")
-        case BillPaymentServices => Messages("businessmatching.registerservices.servicename.lbl.02")
-        case EstateAgentBusinessService => Messages("businessmatching.registerservices.servicename.lbl.03")
-        case HighValueDealing => Messages("businessmatching.registerservices.servicename.lbl.04")
-        case MoneyServiceBusiness => Messages("businessmatching.registerservices.servicename.lbl.05")
-        case TrustAndCompanyServices => Messages("businessmatching.registerservices.servicename.lbl.06")
-        case TelephonePaymentService => Messages("businessmatching.registerservices.servicename.lbl.07")
+  private def businessTypes(activities: BusinessMatching): Option[List[String]] = {
+    val typesString: Option[List[String]] = activities.createBusinessTypeOrderedList
+
+    typesString.map {
+      case t =>
```

Comment:
I think this means we can use flatMap?

Reply:
We lose some of the flexibility of having the option in this case. We have renamed the case variable however.